mkdocs/docs/how-to-release.md

```diff
@@ -208,6 +208,15 @@ svn add /tmp/iceberg-dist-release/
 svn ci -m "PyIceberg <VERSION>" /tmp/iceberg-dist-release/
```

Comment:
I just realized that today I used a slightly different way to copy artifacts to the release dist.

```bash
export RC=rc2
export VERSION=0.7.0${RC}
export VERSION_WITHOUT_RC=${VERSION/rc?/}

export SVN_DEV_DIR_VERSIONED="https://dist.apache.org/repos/dist/dev/iceberg/pyiceberg-${VERSION}"
export SVN_RELEASE_DIR_VERSIONED="https://dist.apache.org/repos/dist/release/iceberg/pyiceberg-${VERSION_WITHOUT_RC}"

svn mv ${SVN_DEV_DIR_VERSIONED} ${SVN_RELEASE_DIR_VERSIONED} -m "PyIceberg: Add release ${VERSION_WITHOUT_RC}"
```
The above commands are derived from https://github.com/apache/iceberg/pull/9926.

If you are going to make any changes to this PR, would you mind also updating the copying to artifacts examples? Thanks in advance! Otherwise, I can update this in a separate PR. : )

Reply:
Nice! Just to confirm: `svn mv` will both move the file, and commit both delete and add actions is that correct?

Updated the docs with your suggestion @HonahX - thank you!

Reply:
Thanks! 

> svn mv will both move the file, and commit both delete and add actions is that correct?
Yes, for example in this case, the https://dist.apache.org/repos/dist/dev/iceberg/pyiceberg-0.7.0rc2 was removed and https://dist.apache.org/repos/dist/release/iceberg/pyiceberg-0.7.0 was added in one svn commit